packages/nhsuk-frontend/src/nhsuk/components/label/template.njk

```diff
@@ -1,18 +1,46 @@
-{% from "nhsuk/macros/attributes.njk" import nhsukAttributes -%}
+{% from "nhsuk/macros/attributes.njk" import nhsukAttributes %}
 
-{% if params.html or params.text %}
+{#- Set classes for this component #}
+{%- set classNames = "nhsuk-label" -%}
+
+{%- if params.classes %}
+  {% set classNames = classNames ~ " " ~ params.classes %}
+{% endif %}
+
+{%- if params.size and params.size in ["s", "m", "l", "xl"] and "nhsuk-label--" not in classNames %}
+  {% set classNames = classNames ~ " nhsuk-label--" ~ params.size %}
+{% endif %}
```

Comment:
I think these should be in the opposite order, so that `classes` are appended last? 

(that gives them higher priority, so that if you specified `size: 'xl'` and `classes: 'nhsuk-label--l'` then it'd render as large)

I think we should also drop `"nhsuk-label--" not in classNames` as that’d false-positive if you had another modifier like `nhsuk-label--somethingelse`?